client/src/pages/course/schedule.tsx

```diff
@@ -119,10 +112,10 @@ const getDefaultViewMode = (): string => {
   }
 
   if (isMobileOnly) {
-    return 'List';
+    return 'list';
```

Comment:
also it would be nice to move these values to constants or to enum, just to simplify editing(you'll have to change it only in one place instead of going through the entire app)